source/2019-08-09-the-ember-times-issue-110.md

```diff
@@ -36,75 +37,79 @@ responsive: true
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Learn All About JAM Stacks Powered By Ember 🍓](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780)
+
+
+There are many different roads that lead developers to building fast, secure and performant websites. And one of them is by the means of the **JAM Stack**.
+
+A site built on the JAMstack is often consisting of a mix of **J**avaScript, reusable and/or third-party **A**PIs as well as static, ready-to-render **m**arkup. The one obligatory feature of a JAMstack site, is that it is can be run without a dedicated web server. Common examples of tools that allow you build JAMstack sites are static site generators like Jekyll, Hugo, GatsbyJS among many others.
```

Comment:
Maybe for clarity:
```suggestion
A site built on the JAMstack combines three things: (1) **J**avaScript, (2) reusable and/or third-party **A**PIs, and (3) static, ready-to-render **m**arkup. The one obligatory feature of a JAMstack site is that it can be run without a dedicated web server. Common examples of tools that allow you to build JAMstack sites are static site generators like Jekyll, Hugo, and GatsbyJS.
```